docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -85,12 +85,12 @@ It is common for datasets to have [outliers](https://en.wikipedia.org/wiki/Outli
 
 In the map below, as you toggle between styles, you will notice how **Style 4** and **Style 5** change based on the modifications made to second (minimum) and third (maximum) parameters.
 
-```js
+```CARTOVL_Viz
```

Comment:
@makella I think this should be `CARTO_VL_Viz` in all cases. 

I think that will make no difference now, as we don't have a specific highlight yet, but it could in the future. Right @davidmanzanares ?